src/cljs/com/github/discoverAI/snake/endpoint.cljs

```diff
@@ -0,0 +1,22 @@
+(ns com.github.discoverAI.snake.endpoint
```

Comment:
this is not really an endpoint but an api dependent functions

Reply:
i would like to name this the same as the namespace in the backend. then we can use namespaced keywords ::key-pressed, because the namespaces match